### ISSUE TITLE
Webchat not working

### ISSUE THREAD

**Issue by Valentinez** [webbukkit/DynmapForge#81]
If I set:

`use-player-login-ip: true`

I get a 500 internal server error, but nothing is showing in my apache log...

Says its coming from line 9666 of jquery-1.11.0.js:

`xhr.send( ( options.hasContent && options.data ) || null );`

I get same error if I try to enable login, and register.
If I hide IP, or just let usernames be the ip, works fine.

**Comment by caasi** [comment]
I send the payload with curl and get the same error in the `1.12.1` branch.

**Comment by caasi** [comment]
I found a fix: https://github.com/AlmuraDev/DynmapForge/commit/60bbea3a3e54210f5a9de90f5203d7372c195380

Check this issue for more information: https://github.com/webbukkit/DynmapForge/issues/62#issuecomment-374835180

**Comment by caasi** [comment]
I think @haffmans [fixed this issue](https://github.com/webbukkit/DynmapForge/pull/77) in 1.12.2. I will test his solution in 1.12.1 later.

**Comment by mikeprimm** [comment]
Yep are you running 3.0-alpha-3, or are you still on an earlier version?  The fix should be in the latest 2.6 builds (which is the stable branch), and should have made it in to the 2.6-beta-1 release.  Latest 2.6 dev builds are here (for forge - http://dynmap.us/builds/DynmapForge/) - I'd suggest 3.0-alpha-3, though.

**Comment by caasi** [comment]
My friends' server stucks at the version 1.12.1. The `Dynmap-2.6-beta-1-forge-1.12.jar` build can be loaded but the issue is still there without the commit b0d1280. They just set `use-player-login-ip` to `false` to avoid it.

I wish they can upgrade to v3.0 so we will be benefit from the DynmapBlockScan, but it's not my call.
